src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,23 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<Props> = ({
+  comments,
+}) => (
+  <div className="CommentList">
+    {comments.map((comment) => {
+      return (
+        <CommentInfo
+          comment={comment}
+          key={comment.id}
+        />
+      );
```

Comment:
Wrap if you have > 2 params or length > 80
```suggestion
    {comments.map((comment) => (
        <CommentInfo comment={comment} key={comment.id} />
     );
```

Reply:
fixed